iree/compiler/Dialect/Flow/Transforms/DispatchConfig.h

```diff
@@ -41,6 +42,12 @@ class OpDispatchPolicy {
   OpDispatchPolicy(Dispatchability &dispatchability)
       : dispatchability(dispatchability) {}
 
+  // Returns true if |op| is not able to fuse with either producer or consumer.
+  static bool isUnsupportedFusionOp(Operation *op);
+
+  // Returns true if |op| can only be a root op.
+  static bool canOnlyBeRootOp(Operation *op);
```

Comment:
nit: `isRootOnlyOp` to match the other methods (and likely what trait names would be, like a `RootOnly` trait

Reply:
done